src/parse_execution.cpp

```diff
@@ -848,7 +848,7 @@ end_execution_reason_t parse_execution_context_t::populate_plain_process(
     wcstring path_to_external_command;
     if (process_type == process_type_t::external || process_type == process_type_t::exec) {
         // Determine the actual command. This may be an implicit cd.
-        bool has_command = path_get_path(cmd, &path_to_external_command, parser->vars());
+        bool has_command = path_get_path(cmd, &path_to_external_command, parser->vars(), true);
```

Comment:
Why does this use the absolute path?

Reply:
:facepalm: It was meant be be false everywhere apart from `parse_cmd_string`. I put it there by mistake and didn't check it after. Sorry.